apps/frontend-manage/src/pages/index.tsx

```diff
@@ -124,22 +151,66 @@ function Index() {
                     className="w-full h-full mt-4 md:overflow-y-auto md:mx-auto"
                     key="question-list"
                   >
-                    <Button
-                      onClick={() =>
-                        setIsQuestionCreationModalOpen(
-                          !isQuestionCreationModalOpen
-                        )
-                      }
-                      className={{
-                        root: twMerge(
-                          'float-right mb-3 font-bold text-white',
-                          theme.primaryBgDark
-                        ),
-                      }}
-                      data={{ cy: 'create-question' }}
-                    >
-                      FRAGE ERSTELLEN
-                    </Button>
+                    <div className="flex flex-row content-center justify-between w-full h-full mt-4 pl-7">
+                      <div className="relative flex flex-row w-2.5/5 pb-6">
+                        <TextField
+                          placeholder="Suchen.."
+                          value={searchInput}
+                          onChange={(newValue: string) => {
+                            setSearchInput(newValue)
+                            handleSearch(newValue)
+                          }}
+                          icon={faMagnifyingGlass}
+                          className={{
+                            input: 'h-10',
+                            field: 'w-1/2',
+                          }}
+                        />
+                        <div className="flex flex-row justify-center w-1/2 ml-1">
+                          <Button
```

Comment:
This button seems to shift backwards and forwards, depending on whether it is disabled or not?

<img width="510" alt="Screenshot 2023-01-27 at 17 10 52" src="https://user-images.githubusercontent.com/80708107/215138029-a4df2b44-f15d-4093-8dc5-adec440c9c3f.png">
<img width="415" alt="Screenshot 2023-01-27 at 17 10 37" src="https://user-images.githubusercontent.com/80708107/215138039-da1586f5-0b12-4e48-9ee1-f225a9739e48.png">

Reply:
is fixed